### ISSUE TITLE
Disable payment channel manager by default

### ISSUE THREAD

**Issue by rvagg** [filecoin-project/lotus#12718]
On node startup, even for a light node, the payment channel manager starts up and watches the chain for paych-related messages to react to. Payment channels get minimal (to no) use currently on mainnet so we should be safe to introduce a configuration option to disable the payment channel manager and turn it on by default.

See https://github.com/filecoin-project/lotus/blob/0d1d7c29c21b41de79be67ef4dcedb4f973bf7c2/node/builder_chain.go#L120-L126

**Comment by rvagg** [comment]
Yeah, sorry, bad wording - turn it off by default, stop it humming in the background.

A few of us are also discussing how we might trim the code down around the paych functionality, removing most, or even all of it from Lotus.

**Comment by ameeetgaikwad** [comment]
is anyone working on this?

**Comment by rvagg** [comment]
Nobody is working on it; you're welcome to take a stab at this @ameeetgaikwad, it might be interesting for you dig through and find all the ways that it's plumbed into a running node. You might want to put some debug logging (just a printf would do it) in the payment channels manager to see it running when you run a node (e.g. in an itest), and then you should be able to see that go away when you stop it running. I expect at least the rate limiting tests in the gateway itest to fail when it's disabled because it takes into account the chatty manager that keeps on polling chain state.

@dd45e640b42e6da7da96faee3996ef7c the proposal here is to simply disable something that isn't being used, it's not a critique of payment channels; in fact the critique of payment channels _is_ that they aren't being used. The original design goal was good—we want to enable easy payments and everyone thinks that getting more FIL moving for payments is positive, but it seems that payment channels were too complex, or were simply the wrong fit.

The next iteration of getting payments to be simpler is probably going to be FWS, you can see the background in this video: https://www.youtube.com/watch?v=Yz5-9oP9d0Q and right now there's a ton of work going in to making this a reality. There's even active work on a payments facilitator you can look at: https://github.com/FilOzone/fws-payments
